package.json

```diff
@@ -47,5 +47,6 @@
     "preprocess": "rm -rf ./bootloader/build && yarn ts-node scripts/process.ts",
     "deploy-preimages": "ts-node scripts/deploy-preimages.ts",
     "compile-yul": "ts-node scripts/compile-yul.ts"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
are you sure this change is needed?

Reply:
removed for now. But we'll need a way to 'align' the yarn versions from zksync-era and here.